<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChoseComis</class>
 <widget class="QDialog" name="DialogChoseComis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить комиссию</string>
  </property>
  <widget class="QLabel" name="lab_year">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Год</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_year">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_ruk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Руководитель</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_ruk">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_comis">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_podl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Подл. ч. к.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_podl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
